### PR TITLE
JENKINS-52225 Upgrade to latest JSch Version

### PR DESCRIPTION
**Pull request by nrayapati**
# Description
- Upgraded to latest JSch Version
- Update all DisplayNames to include the step name
- Add log message to sshScriptStep to print the file name.

See [JENKINS-52225](https://issues.jenkins-ci.org/browse/JENKINS-52225).

### CODE REVIEW

**Comment by nrayapati** [comment]
Have tested this locally and changes looks good. Thank you.

File: src/main/java/org/jenkinsci/plugins/sshsteps/steps/CommandStep.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public String getFunctionName() {
 
     @Override
     public String getDisplayName() {
-      return getPrefix() + "Execute command on remote node.";
+      return getPrefix() + "sshCommand - Execute command on remote node.";
```
Comment (ghenkes) [inline src/main/java/org/jenkinsci/plugins/sshsteps/steps/CommandStep.java]: Can this use `getFunctionName`?

Comment (nrayapati) [reply src/main/java/org/jenkinsci/plugins/sshsteps/steps/CommandStep.java]: Nice catch! Updated with https://github.com/jenkinsci/ssh-steps-plugin/pull/2/commits/a89f5da1a19bd2473493836356fae3d02e795c1f

**Comment by nrayapati** [comment]
Thank you @ghenkes

**Comment by nrayapati** [comment]
Thank you @wuchenwang 👍

**Pull request merged**